src/org/swiftsuspenders/Injector.hx

```diff
@@ -453,7 +454,13 @@ class Injector extends EventDispatcher
 		#else
 		var type:Class<Dynamic> = _reflector.getClass(target);
 		#end
-		
+
+		var injectionPoint: InjectionPoint = _classDescriptor.getDescription(type).injectionPoints;
+		while (injectionPoint != null)
+		{
+			injectionPoint = injectionPoint.next;
```

Comment:
Why are you doing this here? I don't see any usage of local var `injectionPoint`

Reply:
Removed!